lib/archethic/bootstrap.ex

```diff
@@ -209,13 +211,60 @@ defmodule Archethic.Bootstrap do
     end
 
     Archethic.Bootstrap.NetworkConstraints.persist_genesis_address()
+
+    if P2P.authorized_node?() && P2P.available_node?() do
+      resync_network_chain([:oracle, :node_shared_secrets], P2P.authorized_and_available_nodes())
```

Comment:
You should maybe evict yourself, as you are authorized, so to avoid stale download you should target only remote nodes